application/event_processor/changes.py

```diff
@@ -10,7 +10,7 @@
     PUBLICNAME_CHANGE_KEY,
     WEBSITE_CHANGE_KEY,
 )
-from dos import DoSService
+from dos import DoSService, valid_dos_postcode
```

Comment:
Will do

Reply:
@rohith1122 I have changed the function functionality. It will now be get_valid_dos_postcode() because it's now needed to include retrieval of a postcode from the DoS database. (see **https://github.com/nhsd-exeter/dos-integration/pull/55#discussion_r768834044**)

It also implicitly validates the postcode as a NoneType is returned if there are no matching records.

Hopefully this also solves the issue also.

Reply:
Thanks Sam